autogpt_platform/backend/backend/executor/database.py

```diff
@@ -74,6 +78,7 @@ def send_execution_update(self, execution_result: ExecutionResult):
 
     # Credits
     spend_credits = exposed_run_and_wait(_spend_credits)
+    top_up_intent = exposed_run_and_wait(_top_up_intent)
```

Comment:
I don't see where this function is used.

Reply:
Twas removed because the button was swapped to billing. Good spot